.github/workflows/cicd.yml

```
@@ -1,15 +1,34 @@
 name: CI/CD pipeline
 
 on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)